samples/snippets/csharp/VS_Snippets_Misc/tpl_childtasks/cs/child1a.cs

```diff
@@ -10,17 +10,17 @@ public static void Main()
       var parent = Task.Run(() => {
             Console.WriteLine("Parent task executing.");
             var child = Task.Factory.StartNew(() => {
-                  Console.WriteLine("Child starting.");
+                  Console.WriteLine("Attached child starting.");
                   Thread.SpinWait(5000000);
-                  Console.WriteLine("Child completing.");
+                  Console.WriteLine("Attached child completing.");
```

Comment:
I think "completed" sounds more correct than "completing"

```suggestion
                  Console.WriteLine("Attached child completed.");
```

Reply:
I would prefer leaving this unchanged. The `WriteLine` call is inside the child thread. For that reason, it hasn't completed when this is written. (If it were written in the parent thread, "completed" would be better.